### ISSUE TITLE
Having this issue on pop os 21.04 when I run the FAHControl command

### ISSUE THREAD

**Issue by valkyrie-system** [fcole90/fah-snap#16]
folding-at-home-fcole90.FAHControl
Gtk-Message: 03:36:16.113: Failed to load module "appmenu-gtk-module"

(FAHControl:58263): dbind-WARNING **: 03:36:16.116: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-r0YBeyPc9D: No such file or directory

**Comment by valkyrie-system** [comment]
It's working after a fresh install of pop 21.10 (didn't fresh install just to use this, a different thing happened with my OS install)
